src/framework/parsers/glb-parser.js

```diff
@@ -1108,6 +1108,28 @@ const extensionIridescence = (data, material, textures) => {
     }
 };
 
+const extensionAnisotropy = (data, material, textures) => {
+
+    material.enableGGXSpecular = true;
+
+    if (data.hasOwnProperty('anisotropyStrength')) {
+        material.anisotropy = data.anisotropyStrength;
```

Comment:
Should this set anisotropyIntensity, instead of going via the deprecated API?

Reply:
Fixed at https://github.com/playcanvas/engine/pull/7668/commits/bc2cc20072dd2189de26b7da5ba033f4932714f7